libs/net/dal/Migrations/TNOContextModelSnapshot.cs

```diff
@@ -569,12 +569,15 @@ protected override void BuildModel(ModelBuilder modelBuilder)
 
             modelBuilder.Entity("TNO.Entities.CBRAReportStaffSummary", b =>
                 {
-                    b.Property<decimal>("cbra_hours")
-                        .HasColumnType("numeric");
+                    b.Property<string>("CbraHours")
```

Comment:
This looks like an issue.

Reply:
I will take a look. these names are generated automatically.